arm/Microsoft.AppConfiguration/configurationStores/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,55 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
```

Comment:
Not (yet) removed. Please add reference to `deploy.bicep`.

Also, make sure the roles are up to date. If you have question on how - let me know.

Reply:
> Please add a status badge for the corresponding successful pipeline in your fork (Ref https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Contribution%20flow#open-a-pr)

I added the status badge to the top of the PR, is there anywhere else I need to place it?

Reply:
Only in the PR as you did. Overall looks good to me. @MrMCake any additional thoughts?